core/steam_client.gd

```diff
@@ -6,6 +6,7 @@ extends Node
 ## [InteractiveProcess] to spawn steamcmd in a psuedo terminal to read and write 
 ## to its stdout/stdin.
 
+const steam_url := "https://steamdeck-packages.steamos.cloud/archlinux-mirror/jupiter-main/os/x86_64/steam-jupiter-stable-1.0.0.81-2.5-x86_64.pkg.tar.zst"
```

Comment:
This seems fragile, perhaps it would be better to find the highest version available in the webdir?